samples/star/src/androidMain/kotlin/com/slack/circuit/star/imageviewer/ImageViewerScreen.kt

```diff
@@ -118,7 +122,17 @@ fun ImageViewer(state: State, modifier: Modifier = Modifier) {
                 .apply { state.placeholderKey?.let(::placeholderMemoryCacheKey) }
                 .build(),
             contentDescription = "TODO",
-            modifier = Modifier.fillMaxSize(),
+            modifier = Modifier.fillMaxSize()
+              .sharedBounds(
+                sharedContentState = rememberSharedContentState(key = "animal-${state.id}"),
+                animatedVisibilityScope = sharedElementAnimatedContentScope(),
+              )
+              .sharedElement(
+                state = rememberSharedContentState(key = "animal-image-${state.id}"),
+                animatedVisibilityScope = sharedElementAnimatedContentScope(),
```

Comment:
Is there a particular reason why you are adding both sharedBounds and sharedElement on this item?

Reply:
Nope! Was moving them around and alternating between the two.